build/dist

```diff
@@ -247,18 +247,25 @@ BUILD_COMMAND=("$MVN" clean install $MVN_DIST_OPT $@)
 echo -e "\nBuilding with..."
 # shellcheck disable=SC2145
 echo -e "\$ ${BUILD_COMMAND[@]}\n"
+"${BUILD_COMMAND[@]}"
 
+FILTERED_ARGS=()
+# shellcheck disable=SC2045
+for arg in "$@"; do
```

Comment:
The logic can be a function so that other modules/logics do not need to rely on the context of the script.